garak/resources/fixer/20250423_specialwords.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from garak.resources.fixer import Migration
+from garak.resources.fixer import _plugin
+
+
+# commented out pending required functionality in fixer
+"""
+class RenameSpecialwords(Migration):
+    def apply(config_dict: dict) -> dict:
+        # Rename ex-specialwords detectors to their landing places
+
+        path = ["plugins", "detectors"]
+        renames = (
+            ["specialwords.SlursReclaimedSlurs", "unsafe_content.SlursReclaimedSlurs"],
+            ["specialwords.Prefixes", "mitigation.Prefixes"],
+        )
+        updated_config = config_dict
+        for old, new in renames:
+            updated_config = _plugin.rename(updated_config, path, old, new)
```

Comment:
The rename support full module or individual class rename, not distribution to other existing modules, the configuration adjustments will have to be full implemented by this class for this case.  A new helper could be created if the pattern is determined to be common.

Reply:
tracked in #1222